source/concepts/aggregate-transaction.rst

```diff
@@ -6,23 +6,53 @@ Aggregate Transaction
 
 Aggregated Transactions merge multiple transactions into one, allowing **trustless swaps**, and other advanced logic. NEM does this by generating a one-time disposable smart contract. When all involved :doc:`accounts<../concepts/account>` have cosigned the aggregate transaction, all the inner transactions are executed at the same time.
 
-********
-Examples
-********
+.. _aggregate-complete:
+
+******************
+Aggregate complete
+******************
+
+An aggregate transaction is  **complete** when all cosigners (for multisignature account) have signed it.
+
+The cosigners can sign the transaction without using the blockchain (offline through their own localhost). Once it has all the required signatures, one of them can announce it to the network. If the inner transaction setup is valid, and there is no validation error, the transactions will get executed at the same time.
+
+Aggregate complete transactions enable adding more transactions per block by gathering multiple inner transactions.
+
+.. note:: Aggregate complete transaction has only one signing account. An aggregate transaction can involve different types of transaction. 
```

Comment:
I suggest deleting this note since it can have multiple signers.

Reply:
What I was trying to say is that there would be only one account that is announcing the transaction, however, that account can be a multisig account that need more than one signer. As Aggregate complete is a one to many, one direction transaction. Correct me if I am wrong. 
Couldn't find a best way to describe that yet.